playfab-docs/features/multiplayer/servers/quickstart-for-multiplayer-servers-api-powershell.md

```diff
@@ -91,7 +100,7 @@ Once we see some standing by servers, let's request one for gameplay...
 $Regions = new-object 'System.Collections.Generic.List[PlayFab.MultiplayerModels.AzureRegion]'
 $Regions.Add("EastUS");
 
-$svr = New-PFMultiplayerServer -BuildName "MyBuild" -SessionId "00000000-0000-0000-0000-000000000001" -SessionCookie "test cookie" -PreferredRegions $regions
+$svr = New-PFMultiplayerServer -BuildName "PSTest_built" -SessionId "00000000-0000-0000-0000-000000000001" -SessionCookie "test cookie" -PreferredRegions $regions
```

Comment:
Nit: should it be PSTest_build

Reply:
Yeah. I'll change the other instance that was already misspelled.